lib/src/main/java/org/buffer/android/counterview/CounterView.kt

```diff
@@ -40,8 +41,10 @@ class CounterView : AppCompatTextView {
             updateCounterValue(textView?.length() ?: 0)
         }
 
-    private var counterTextColor = -1
-    private var counterErrorTextColor = -1
+    @ColorInt
+    var counterTextColor = -1
```

Comment:
any reason why this has been made public? If you set this publicly at any point, then no change will be reflected

Reply:
This was made public so it could be set programmatically. You're right, the view wont reflect it until the text in the edittext changes, at which point the color will be updated in `updateCounterValue`. I think a better approach might be to make these private again, and have custom setters that updates the textview color when it's set. @hitherejoe What do you think about that approach?

Reply:
@vgonda yep that would be the correct way to do it, that way any external caller won't have direct access to the field and we can udate the UI accordingly. Having the view update when the field is set it a more reliable approach than relying on the view updating during `updateCounterValue` 👍

Reply:
Makes sense! :)